framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -233,6 +243,16 @@
     /** Parses stub files and stores annotations from stub files. */
     public final AnnotationFileElementTypes stubTypes;
```

Comment:
Should the documentation of these two fields mention that they only store public elements?